run/authentication/auth.js

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START run_service_to_service_auth]
+// Make sure to `npm install gcp-metadata` and `npm install got` or add the dependencies to your package.json
+const gcpMetadata = require('gcp-metadata')
+const got = require('got');
+
+const requestServiceToken = async () => { 
+  try {
+
+    // Add the URL of your receiving service
+    const receivingServiceURL = ...
```

Comment:
For testing purpose we should pass this in as a func argument. You can add a TODO to uncomment and set it like https://github.com/googleapis/nodejs-dlp/blob/master/samples/risk.js#L55